src/frontend/next/src/components/SEO.tsx

```diff
@@ -0,0 +1,47 @@
+import { Helmet } from 'react-helmet';
+import useSiteMetadata from '../hooks/use-site-metadata';
+
+type SEOProps = {
+  description?: string;
+  lang?: string;
+  meta?: Array<any>;
+  title?: string;
+};
+const SEO = ({ description, lang, meta, title }: SEOProps) => {
+  const siteMetadata = useSiteMetadata();
+  const metaDescription = description || siteMetadata.description;
+  const concatMeta = [
+    {
+      name: `description`,
+      content: metaDescription,
+    },
+    {
+      property: `og:title`,
+      content: title,
+    },
+    {
+      property: `og:description`,
+      content: metaDescription,
+    },
+  ].concat(meta ?? []);
```

Comment:
What in the world

Reply:
Similar to doing `meta || []` but more type specific.  With `||` you are looking for a "falsey" value on the left, whereas with `??` you are looking for null or undefined.

I've rarely seen it used, but that's what it's for.  @tonyvugithub any particular reason why you chose it here?